administration/src/bp-modules/applications/PreVerifiedEntitlementType.tsx

```diff
@@ -0,0 +1,34 @@
+import { GeneralJsonField, type JsonField, findValue } from './JsonFieldView'
+
+export const enum PreVerifiedEntitlementType {
+  Juleica = 'blueCardJuleicaEntitlement',
+  Verein360 = 'blueCardWorkAtOrganizationsEntitlement',
+  HonoredByMinisterPresident = 'goldenCardHonoredByMinisterPresidentEntitlement',
+}
+
+const hasPreVerifiedEntitlement = (
+  applicationDetails: JsonField<'Array'>,
+  field: PreVerifiedEntitlementType
+): boolean => {
+  const entitlement = findValue(applicationDetails, field, 'Array')
+  if (field === PreVerifiedEntitlementType.Verein360) {
+    const entitlements = entitlement?.value ?? []
+    return entitlements.some(
+      (item: GeneralJsonField) =>
+        Array.isArray(item.value) &&
+        item.value.some((subItem: GeneralJsonField) => subItem.name === 'isAlreadyVerified' && subItem.value === true)
+    )
+  }
+  return !!entitlement
```

Comment:
🔧 I would prefer a more explicit form like `entitlement != undefined` here, as this better conveys the actual test. The double bang relies on JavaScript's implicit type conversion and this should be avoided like the plague.